sys/src/cmd/screenlock.c

```diff
@@ -241,21 +240,23 @@ lockscreen(void)
 	procrfork(blanker, nil, 4096, RFFDG);
 	fd = open(pic, OREAD);
 	if(fd > 0){
+		draw(screen, screen->r, display->black, nil, ZP);
```

Comment:
Do you really want to draw a black box here before you know the readimage succeeded?